Darwin/flash

```diff
@@ -97,8 +97,8 @@ filename=$(basename "${image}")
 extension="${filename##*.}"
 filename="${filename%.*}"
 
-if [ -f "/tmp/${filename}.${extension}" ]; then
-  image=/tmp/${filename}.${extension}
+if [ -f "/tmp/${filename}" ]; then
```

Comment:
Ok what happens my machine is that extension gets set to .zip which is correct and the filename gets set to biglongname.img

However when you run the unzip, the .zip file is deleted, so when it does the check, it always does the download over and over. Could be because there is another conflict with gnu tools. I could not see where you delete the zip file, do you know which of your downstream commands does this?

Thanks
Rich

—
Never give in. Never, never, never, never, in nothing great or small, large or petty, never give in except to convictions of honor and good sense. Winston Churchill, Harrow School, 29 Oct 41
---
tongfamily.com | igncap.com | fb.me/richtong | @richtong | +1-206-963-9029 | +86-137-6184-9029

> On Mar 16, 2016, at 3:57 PM, Stefan Scherer <notifications@github.com> wrote:
> 
> In Darwin/flash <https://github.com/hypriot/flash/pull/46#discussion_r56430512>:
> 
> > @@ -97,8 +97,8 @@ filename=$(basename "${image}")
> >  extension="${filename##*.}"
> >  filename="${filename%.*}"
> >  
> > -if [ -f "/tmp/${filename}.${extension}" ]; then
> > -  image=/tmp/${filename}.${extension}
> > +if [ -f "/tmp/${filename}" ]; then
> This reverts surround-io@859e0cb <https://github.com/surround-io/flash/commit/859e0cbb800e6d097faa5ddf39bbd18f27973553>
> I think I've changed that while building new zipped image files locally and tested and build and tested again.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/hypriot/flash/pull/46/files/78241d753719870e08f913d0f93e8790d805192d#r56430512>